shell/linters/prettier.sh

```diff
@@ -11,13 +11,13 @@ extensions=(yaml yml json md ts)
 
 prettier_linter() {
   yarn_install_if_needed >/dev/null
-  find_files_with_extensions "${extensions[@]}" | xargs -n40 "node_modules/.bin/prettier" -l --loglevel log
```

Comment:
`node_modules/.bin` isn't a thing in `pnpm` linking. `yarn run <script>` has always supported doing this for us, so this is more clean this way anyways.